arc_solvers/models/qa/README.md

```diff
@@ -0,0 +1,67 @@
+# Models for Question Answering
+This document contains description of the neural models for question answering.
+
+## Multi Choice QA
+
+### BiLSTM Max-out with Question to Choices Max Attention
+The [`QAMultiChoiceMaxAttention`](multi_choice/qa_multi_choice_max_att.py)
+model computes the attention interaction between question and choices
```

Comment:
Same edit as before: "... interaction between the context-encoded representations of the question and the choices."